README.md

```diff
@@ -45,9 +46,11 @@ Please note a passing build status indicates all listed APIs are available since
 * [Shopping](#shopping)
 * [Social](#social)
 * [Sports & Fitness](#sports--fitness)
+* [Text Analysis](#text-analysis)
```

Comment:
Regarding 2 entries in the `Text Analysis` section, I'd prefer if we could find a 3rd service just to keep the rule in standing order, but I understand the hesitation of later having to break these two entries out again if we just lob them into `Development` now. @karllhughes , maybe you could find a third one? 😄

Reply:
https://www.ibm.com/watson/services/natural-language-understanding/ 
https://cloud.google.com/natural-language/ 

🤔

Reply:
That works 💯